### ISSUE TITLE
Interface TX off

### ISSUE THREAD

**Issue by gageorsburn** [kubevirt/kubevirt#1715]
**Is this a BUG REPORT or FEATURE REQUEST?**:
/kind bug

**What happened**:
While booting a cumulus linux VM, dhcp offers were rejected due to checksum failures. Seems pretty similar to https://github.com/projectcalico/felix/issues/40. TX should be turned off on the dhcp interfaces since it shouldn't be needed with virtual interfaces anyway. DHCP worked successfully after ethtool was put inside the virt-launcher pod and the following commands were run:

```
for i in $(kubectl get po -l kubevirt.io=virt-launcher -o json | jq -r '.items[].metadata.name') ; do echo $i ; done
for i in $(kubectl get po -l kubevirt.io=virt-launcher -o json | jq -r '.items[].metadata.name') ; do kubectl cp /tmp/ethtool $i:/tmp/ -c compute ; done
for i in $(kubectl get po -l kubevirt.io=virt-launcher -o json | jq -r '.items[].metadata.name') ; do kubectl exec $i -c compute -- /tmp/ethtool -K k6t-eth0 tx off ; done
for i in $(kubectl get po -l kubevirt.io=virt-launcher -o json | jq -r '.items[].metadata.name') ; do kubectl exec $i -c compute -- /tmp/ethtool -K vnet0 tx off ; done
```

**What you expected to happen**:
Cumulus linux to boot and eth0 to receive passthrough address from dhcp server in pod.

**How to reproduce it (as minimally and precisely as possible)**:
Boot cumulus linux

**Anything else we need to know?**:

**Environment**:
- KubeVirt version (use `virtctl version`):
- Kubernetes version (use `kubectl version`):
- VM or VMI specifications:
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by gageorsburn** [comment]
@booxter @yuvalif Thoughts on this?

**Comment by booxter** [comment]
@gageorsburn seems like the right approach. We could also forcefully fill checksums in iptables rules but that seems excessive and not useful for virtual pod interfaces.

**Comment by SchSeba** [comment]
Related https://github.com/weaveworks/weave/blob/master/net/veth.go#L53 and also https://github.com/gregoryolsen/lxc-devuan/issues/1

**Comment by booxter** [comment]
FTR we also had this issue in OpenStack Neutron when DHCP agent was running on the same node as a VM.

**Comment by analistamarcioprado** [comment]
Good Morning. We are facing the same problem. But we imagine that the problem is not the DHCP itself.

In the Neutral documentation it says: "Computing hosts should always have the appropriate NICs set to promiscuous mode."

This is necessary for the host to receive frames addressed to other MACs (from a VM hosting on it for example). This is required for VMs to be able to respond to an IP offer.

Perhaps this may be more effectively related to DHCP malfunction.

I came to manually configure the IPs on the VMs, and yet the (intermittent) communication problem continued ... Only after several reboots of all cloud hosts did the VMs re-communicate.

**Comment by rmohr** [comment]
@analistamarcioprado I suppose that you have a kubevirt installation which already includes #1766. Would you mind sharing a little bit more info here or in a new issue (like which version you are using)?
